app/templates/partials/footer.html

```diff
@@ -17,7 +17,7 @@
     <script src="{{ url_for('static', filename='light/dist/js/sb-admin-2.js') }}"></script>
    <script>
 
-       document.getElementById("clickGoal").onclick = ldclient.track("chart-click");
+       //document.getElementById("clickGoal").onclick = ldclient.track("chart-click");
```

Comment:
Is this commented out?

Reply:
I may have disabled it, but dont know if it was working in other areas of the app. I wasnt using that code